spec/requests/api/v1/events/nearest_spec.rb

```diff
@@ -0,0 +1,60 @@
+describe 'Events Requests' do
+  describe 'GET /v1/events/nearests?lat=&lon=&radius=' do
+    it 'returns events closest to lat and lon' do
+      near_event = create(:event, :worcester)
+      farther_event = create(:event, :springfield)
+      boston_event = attributes_for(:event, :boston)
+      create(:event, :san_fran)
+      # distance is measured in kilometers (Mass is about 306 km across)
+      radius = 306
+
+      get "/v1/events/nearests?lat=#{boston_event[:lat]}&lon=#{boston_event[:lon]}&radius=#{radius}"
```

Comment:
Should have caught this before, but you should use the rails path helpers. Something like: `nearests_v1_events_path(lat: boston_event[:lat], lon: boston_event[:lon], radius: radius)`